src/pages/Admin/Venue/VenueAdminPage.tsx

```diff
@@ -58,3 +59,12 @@ export const VenueAdminPage: React.FC = () => {
     </>
   );
 };
+
+export const VenueAdminPage = () => {
+  const venueId = useVenueId();
+  return (
+    <RelatedVenuesProvider venueId={venueId}>
+      <VenueAdminPageContents />
+    </RelatedVenuesProvider>
+  );
+};
```

Comment:
Why do we need to extract a new wrapper component here? Does `VenueAdminPage` itself cause the error? Or it's children? If the latter, just include it in the JSX within `VenueAdminPage`.

Reply:
But since we need to make changes to fix things as per https://github.com/sparkletown/sparkle/pull/1769#discussion_r666850256, it would make sense to follow that same pattern and just use that pattern to handle this at the `AppRouter` layer.

Reply:
`VenueAdminPage` uses `useConnectCurrentVenueNG(venueId)`  itself, so it needs provider to wrap it.

Reply:
That hook has nothing to do with the world users provider..?

Reply:
reverted in 5efe4611ba289afe281b83ccf1106504ed793b68